toxcore/Messenger.h

```diff
@@ -65,6 +65,17 @@ typedef struct Messenger_Options {
 } Messenger_Options;
 
 
+enum ToxCapabilitiesFlags {
+    TOX_CAPABILITY_BASIC = 1 << 0,
+    TOX_CAPABILITY_PGC = 1 << 1,
+    TOX_CAPABILITY_H264 = 1 << 2,
+    TOX_CAPABILITY_MESSAGE_V2 = 1 << 3,
+};
+
+#define TOX_CAPABILITIES_CURRENT (uint64_t)(TOX_CAPABILITY_BASIC)
+#define TOX_CAPABILITIES_SIZE sizeof(uint64_t)
```

Comment:
The size of a capabilities packet is 1 + sizeof(uint64_t).

Reply:
but the pure data size of the capabilties is sizeof(uint64_t)